manager/manager.py

```diff
@@ -415,11 +415,18 @@ def insert_job(self, job):
         self.job_db_handler.insert_item(job_data)
 
     def query_jobs(self, data=None):
-        items = self.job_db_handler.query_items(data)
+        items, last_key = self.job_db_handler.query_items(data)
         jobs = []
         if items and len(items):
             for item in items:
                 jobs.append(TxJob(item))
+
+        while last_key != None:
```

Comment:
I'm wondering if we should do all this "last_key" stuff within the query_items() function itself, returning all the items from one call. Otherwise we also have to do this when we list jobs in the list_jobs lambda function if a user by chance has quite a few jobs.